### PR TITLE
Content Card Developer Refresh

### PR DESCRIPTION
**Pull request by josh-mccrowell-braze**

### CODE REVIEW

File: _docs/_developer_guide/customization_guides/content_cards.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+nav_title: Content Cards
+article_title: Content Card Customization
+layout: featured
```
Comment (KellieHawks) [inline _docs/_developer_guide/customization_guides/content_cards.md:4]:
```suggestion
layout: dev_guide
```

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

File: _docs/_developer_guide/customization_guides/content_cards/customizing_behavior.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  - Web
+---
+
+# Customizing Content Card behavior
+
+> This implementation guide discusses changing the behavior of Content Cards, adding extras such as key-value pairs to your payload, and recipes for common customizations.
+
+## Key-value pairs
+
+Braze enables you to send extra data payloads via Content Cards to user devices using key-value pairs. These can help you track internal metrics, update app content, and customize properties. [Add key-value pairs using the dashboard][9].
```
Comment (ttsaiveerappan) [inline _docs/_developer_guide/customization_guides/content_cards/customizing_behavior.md:21]: Can we have this link direct to https://braze-docs-en-git-content-card-refresh-jjm-braze.vercel.app/docs/user_guide/message_building_by_channel/content_cards/create/#step-4-configure-additional-settings-optional in that page

File: _docs/_developer_guide/customization_guides/content_cards/customizing_styles.md [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+For example, to change the default image size and corner radius for all cells:
+
+```swift
+BrazeContentCardUI.ViewController.Attributes.defaults.cellAttributes.cornerRadius = 20
+BrazeContentCardUI.ViewController.Attributes.defaults.cellAttributes.classicImageSize = CGSize(width: 65, height: 65)
+```
+
+**Initializing the view controller with Attributes**
+
+If you wish to modify only a specific instance of the Braze Content Card UI view controller, use the [init(braze:attributes:)](https://braze-inc.github.io/braze-swift-sdk/documentation/brazeui/brazecontentcardui/viewcontroller/init(braze:attributes:)/) initializer to pass a custom `Attributes` struct to the view controller.
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/customization_guides/content_cards/customizing_styles.md:103]: The remainder of similar links in this section use code text, should this be code text as well?
```suggestion
If you wish to modify only a specific instance of the Braze Content Card UI view controller, use the [`init(braze:attributes:)`](https://braze-inc.github.io/braze-swift-sdk/documentation/brazeui/brazecontentcardui/viewcontroller/init(braze:attributes:)/) initializer to pass a custom `Attributes` struct to the view controller.
```

File: _docs/_developer_guide/customization_guides/content_cards/logging_analytics.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+
+All properties outside of `id` and `extras` are optional to parse for custom Content Cards. For more information on the data model, see each platform's integration article: [Android][1], [iOS][2], [Web][3].
+
+
+{% tabs %}
+{% tab Android %}
+{% subtabs local %}
+{% subtab Java %}
+
+### Create a private subscriber variable
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/customization_guides/content_cards/logging_analytics.md:31]: Should these be steps? i.e., Step 1: Create a private subscriber variable

File: _docs/_developer_guide/customization_guides/content_cards/customizing_behavior.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+Content Cards can be leveraged to create dynamic and interactive experiences for your users. In the example to the right, we have a Content Card pop-up appear at checkout providing users last-minute promotions. Well-placed cards like this are a great way to give users a "nudge" toward specific user actions. 
+
+The key-value pairs for this use case include a `discount_percentage` set as the desired discount amount and `class_type` set as `coupon_code`. These key-value pairs allow you to filter and display type-specific Content Cards on the checkout screen. For more information on using key-value pairs to manage multiple feeds, see [Customizing the default Content Card feed][3]. 
+<br>
+<br>
+
+![][5]{: style="max-width:80%;"} 
+
+## Content card badges
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/customization_guides/content_cards/customizing_behavior.md:74]:
```suggestion
## Content Card badges
```

File: _docs/_developer_guide/platform_integration_guides/web/content_cards/integration.md [hunk 6]
```diff
@@ -15,13 +15,64 @@ search_rank: 1
 
 {% multi_lang_include archive/web-v4-rename.md %}
 
+
+The Braze Web SDK includes a Content Cards feed UI to speed up your integration efforts. If you would prefer to build your own UI instead, see the [Content Card Customization Guide]({{site.baseurl}}/developer_guide/customization_guides/content_cards).
+
+### Standard feed UI
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/platform_integration_guides/web/content_cards/integration.md:21]: @josh-mccrowell-braze the heading levels on this page are out of order, which is breaking the TOC

File: _docs/_developer_guide/platform_integration_guides/swift/content_cards/integration.md [hunk 7]
```diff
@@ -11,128 +11,15 @@ channel:
 
 # Content Card integration
```
Comment (bre-fitzgerald) [inline _docs/_developer_guide/platform_integration_guides/swift/content_cards/integration.md:12]: @josh-mccrowell-braze same comment as the web doc, heading levels are out of order

**Review by bre-fitzgerald — COMMENTED** [review]
Looks good! I tested out most of the redirects, which are working for me. Left a few comments for some minor issues/suggestions.

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**